src/features/collapsible-sidebar/__tests__/__snapshots__/CollapsibleSidebarLogo.test.js.snap

```diff
@@ -49,119 +49,128 @@ exports[`components/core/collapsible-sidebar/CollapsibleSidebar render 1`] = `
+              type="button"
             >
-              <IconHamburger
-                className="sc-AykKD kjACFS"
+              <CollapsibleSidebarLogo__StyledIconHamburger
                 height={20}
                 width={20}
               >
-                <AccessibleSVG
-                  className="icon-hamburger sc-AykKD kjACFS"
+                <IconHamburger
+                  className="CollapsibleSidebarLogo__StyledIconHamburger-sc-6amevi-1 jjgzkm"
                   height={20}
-                  viewBox="0 0 24 24"
                   width={20}
                 >
-                  <svg
-                    className="icon-hamburger sc-AykKD kjACFS"
-                    focusable="false"
+                  <AccessibleSVG
+                    className="icon-hamburger CollapsibleSidebarLogo__StyledIconHamburger-sc-6amevi-1 jjgzkm"
                     height={20}
-                    role="presentation"
                     viewBox="0 0 24 24"
                     width={20}
                   >
-                    <path
-                      className="fill-color"
-                      d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
-                      fill="#000"
-                    />
-                    <path
-                      d="M0 0h24v24H0z"
-                      fill="none"
-                    />
-                  </svg>
-                </AccessibleSVG>
-              </IconHamburger>
-            </Styled(IconHamburger)>
-          </button>
-        </PlainButton>
+                    <svg
+                      className="icon-hamburger CollapsibleSidebarLogo__StyledIconHamburger-sc-6amevi-1 jjgzkm"
+                      focusable="false"
+                      height={20}
+                      role="presentation"
+                      viewBox="0 0 24 24"
+                      width={20}
+                    >
+                      <path
+                        className="fill-color"
+                        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
+                        fill="#000"
+                      />
+                      <path
+                        d="M0 0h24v24H0z"
+                        fill="none"
+                      />
+                    </svg>
+                  </AccessibleSVG>
+                </IconHamburger>
+              </CollapsibleSidebarLogo__StyledIconHamburger>
+            </button>
+          </PlainButton>
+        </CollapsibleSidebarLogo__StyledToggleButton>
         <LinkBase
           className="bdl-CollapsibleSidebar-logoLink"
         >
           <a
             className="bdl-CollapsibleSidebar-logoLink"
             href="#"
           >
-            <Styled(BoxLogo)
+            <CollapsibleSidebarLogo__StyledLogo
               className="bdl-CollapsibleSidebar-logoIcon"
-              height={32}
-              width={61}
+              height={42}
```

Comment:
Just confirming that these height and width changes are expected...?

Reply:
I think so. @alexkrolick, can you confirm?

Reply:
yes, because some of the pixels moved into padding & margin instead of height